### PR TITLE
Feat/animations on homepage classname="slidein"

### PR DESCRIPTION
**Pull request by robertbolgeo**
# Description

Created custom animation for use across the site. Updated the tailwind config to include a new "slidein" class. Please use "animate-slidein" when you want to use it. Created a component-wrapper component to apply custom animation to whole component pieces. In order to apply a slidein effect on scroll down, I used an eventListener useEffect.

## Related Issues
- Issue/Request: #234 

## Type of change

Please delete options that are not relevant. (All options may be deleted for minor changes)

- [.] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Reloaded the page and scrolled up and down. Has not been tested outside of Chrome on Mac.

- [ ] Please test on various OSs.
- [ ] Please test on various browsers.

### After:

https://github.com/user-attachments/assets/7d06e424-6c40-43ac-bec5-8a24c93a125f

## Checklist
- [.] I have ensured the code adheres to the coding standards of the project.
- [.] If changes require documentation updates, I have included them.
- [.] I have performed a self-review of my code.
- [.] New members can easily understand this code, and I have added comments for clarity.
- [.] The PR is client-ready (all client-related feedback has been addressed).

## Additional Context or Notes
I think that this could be a bit cleaner, but it's a good step forward. If anyone wants to make it better, please do. :clinking_glasses:

### CODE REVIEW

**Comment by NabbeunNabi** [comment]
@robertbolgeo similar to the other PR as how this includes the other two commits but is separate we can rebase and have only the most recent commit for this. I would approve it but it isn't atomic in that it has all the other changes too. Love the work you've done

**Review by NabbeunNabi — CHANGES_REQUESTED** [review]
Comment added above

**Comment by robertbolgeo** [comment]
This PR has been reduced to tears and to one commit.

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -2,49 +2,54 @@ import Blogs from "../components/homeComponents/Blogs";
...
+  <div className="bg-gradient-to-br from-green-300 to-white font-sans">
+   <div className="w-full m-auto"></div>
+   {images && <HeroCarousel images={images} />}
+   <AnimatedSection>
+    <Slogan />
+   </AnimatedSection>
+   <AnimatedSection>
+    <MissionStatement />
+   </AnimatedSection>
+   {/* <section id="cards" className={"min-h-80 flex flex-col px-4"}>
```
Comment (NabbeunNabi) [inline src/pages/Home.tsx]: What is this doing? If it is unnecessary you can remove it. You could always add it later if necessary

**Review by NabbeunNabi — COMMENTED** [review]
Check the comment @robertbolgeo and also if you could remove the period from the PR title. After that it is good to go.

**Review by NabbeunNabi — APPROVED** [review]
LGTM. SHIP IT 🐵

**Pull request merged**